docker/README.md

```diff
@@ -3,7 +3,7 @@
 ## Build - dockerfiles
 
 ```
-sudo docker build -f {DOCKER_FILE_NAME}.dockerfile -t {DOCKER_IMAGE_NAME} .
+sudo docker build --build-arg {ARG_NAME}={ARG_VALUE} -f {DOCKER_FILE_NAME}.dockerfile -t {DOCKER_IMAGE_NAME} .
```

Comment:
@rrawther can you specify what arg is available? like in this case options are: "PYTORCH_VERSION"

Reply:
Yes. I have added a seperate README under pytorch because this is generic to all dockers